infobip-typescript-generator-extension-api/src/main/java/com/infobip/typescript/validation/Localization.java

```diff
@@ -0,0 +1,17 @@
+package com.infobip.typescript.validation;
+
+public class Localization {
+    public static final String LOCALIZATION_CLASS_NAME = "Localization";
+    public static final String LOCALIZATION_METHOD = "localize";
+    public static final String LOCALIZATION_FILE_NAME = LOCALIZATION_CLASS_NAME + ".ts";
+    public static final String LOCALIZATION_SOURCE_CODE =
+            "export function localize(message, object: object = {}) {format(message, object)};\n" +
```

Comment:
mislim da ti fali return prije format

Reply:
imaš pravo mislim da si našao bug u postojećem TS generatoru, ispravim ga s ovom tvojom sugestijom i dao si mi ideju da napravim malui nadogradnju pa i to ubacim :)

Reply:
done